src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -202,23 +200,23 @@ def test_edit_add_tag(self):
         self.tags.insert(0, 'tag4')
         data = self.get_dict()
         r = self.client.post(self.basic_edit_url, data)
-        eq_(r.status_code, 200)
+        assert r.status_code == 200
 
         result = pq(r.content)('#addon_tags_edit').eq(0).text()
 
-        eq_(result, ', '.join(sorted(self.tags)))
-        eq_((ActivityLog.objects.for_addons(self.addon)
-             .get(action=amo.LOG.ADD_TAG.id)).to_string(),
-            '<a href="/en-US/firefox/tag/tag4">tag4</a> added to '
-            '<a href="/en-US/firefox/addon/test_slug/">new name</a>.')
-        eq_(ActivityLog.objects.filter(
-            action=amo.LOG.ADD_TAG.id).count(), count + 1)
+        assert result == ', '.join(sorted(self.tags))
+        html = ('<a href="/en-US/firefox/tag/tag4">tag4</a> added to '
+                '<a href="/en-US/firefox/addon/test_slug/">new name</a>.')
+        assert (ActivityLog.objects.for_addons(self.addon).get(
```

Comment:
I don't think you need the parens around `ActivityLog.objects.for_addons(self.addon).get(action=amo.LOG.ADD_TAG.id)` to call `to_string` properly